Apps/Database/src/Models/ResourceDelegate.cs

```diff
@@ -66,5 +70,17 @@ public class ResourceDelegate : AuditableEntity
         /// Gets or sets the UserProfile associated with <see cref="ProfileHdid"/>.
         /// </summary>
         public virtual UserProfile UserProfile { get; set; } = null!;
+
+        /// <summary>
+        /// Gets or sets the access for the data sets.
+        /// </summary>
+        [Column(TypeName = "jsonb")]
+        [SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Team decision")]
+        public HashSet<DataSource>? DataSources { get; set; }
+
+        /// <summary>
+        /// Gets the delegate invitations.
+        /// </summary>
+        public ICollection<DelegateInvitation>? DelegateInvitations { get; } = new List<DelegateInvitation>();
```

Comment:
```suggestion
        public virtual ICollection<DelegateInvitation>? DelegateInvitations { get; } = new List<DelegateInvitation>();
```
Navigation properties should include the virtual modifier in case we want to enable lazy loading.